src/pages/MetricDetail.svelte

```diff
@@ -321,7 +321,7 @@
               )}>{pingData.bigquery_table}</a
             >
             and <AuthenticatedLink
-              href={`https://mozilla.acryl.io/dataset/urn:li:dataset:(urn:li:dataPlatform:bigquery,moz-fx-data-shared-prod.${params.app}.${selectedPingVariant.id},PROD)`}
+              href={`https://mozilla.acryl.io/dataset/urn:li:dataset:(urn:li:dataPlatform:bigquery,mozdata.${params.app}.${selectedPingVariant.id},PROD)`}
```

Comment:
Seems like the upstream table(s) would have the most complete picture of lineage here